src/App/components/NavigationBar/NavigationBar.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import { BottomNavigation, BottomNavigationAction } from '@material-ui/core';
+
+import RecentActorsIcon from '@material-ui/icons/RecentActors';
+import SelfImprovementIcon from '@material-ui/icons/SelfImprovement';
+
+const StyledFooter = styled(BottomNavigation)`
+  position: fixed;
+  width: 100%;
+  bottom: 0;
+`;
+
+const NavigationBar = () => {
+  const [value, setValue] = React.useState(0);
```

Comment:
As I commented before, use useRouter and set state according to route state

Reply:
```suggestion
  
  const router = useRouter();
  let activeLink = router.pathname === '/practices' ? 1 : 0;
  const [value, setValue] = React.useState(activeLink);

  const handleChange = () => {
    if (router.pathname === '/practices') {
      setValue(prev => prev - 1);
    } else {
      setValue(prev => prev + 1);
    }
  };
```
Can it be done this way?